python/cudf/cudf/core/indexed_frame.py

```diff
@@ -3346,7 +3346,7 @@ def sample(
         frac=None,
         replace=False,
         weights=None,
-        random_state=None,
+        random_state=1,
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Similarly here, I don't think we should set a specific seed as a default argument to sample. This is also creating a difference in the default API wrt pandas (which defaults to None https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.sample.html)